public/app/features/logs/components/LogRows.tsx

```diff
@@ -130,6 +143,7 @@ class UnThemedLogRows extends PureComponent<Props, State> {
                 getRowContext={getRowContext}
                 row={row}
                 showContextToggle={showContextToggle}
+                showRowMenu={!this.state.contextIsOpen}
```

Comment:
On line 117 we are doing `const { renderAll } = this.state;`, so we could just add `contextIsOpen` there as well.

Reply:
Done in 24a19dde06

Reply:
Done in 24a19dde06b811b66847c27f0a3e7ab9500ca314